ExeterTeam25/group-software-project
# Styling create challenge

To test: 
- Run code as normal, go to create a challenge. 
- Style should look better and similar to login page, but work the same. 

What I did: 
- Wrote CSS for the create a challenge page.
